src/modules/wreck/lua.d/timeout.lua

```diff
@@ -0,0 +1,40 @@
+--
+-- Register timer on nodeid 0 if kvs `walltime` is set for this job.
+--  Kill job on timeout if reached.
+--
+
+local function start_timer (f, wreck, id, walltime)
+   local to, err = f:timer {
+        timeout = walltime*1000,
+        oneshot = true,
+        handler = function (f, to)
+          wreck:log_msg ("Timeout expired! Killing job")
+          local rc,err = f:sendevent ({signal = 14}, "wreck.%d.kill", id)
```

Comment:
I think you're right on the signal name thing. It is pretty simple to load posix module anyway. I'll push a change for your review (maybe this evening)

For SIGALRM vs SIGTERM, maybe at this point (since this is still in the prototype phase) we could just make it configurable?